app/javascript/utils/reactFormUtils.js

```diff
@@ -0,0 +1,89 @@
+// Shamles copy paste from react-form/redux/utils, Fed
```

Comment:
@fddayan Seems like lodash already offers most of these: https://lodash.com/docs/4.17.10. Can you replace any usage of these with lodash methods where possible, keeping just the ones that are not offered by lodash? These are totally fine, but it's less complicated to stick to one set of util functions as much as possible.

Reply:
I tried replacing the lodash methods `isArray`, `isObject`, `flattenDeep` and validations stopped working. There might be subtle differences.I did not dig into why.

Reply:
@callaghanc as we talked on slack. I added a NOTE and TODO comment.

Reply:
🙏